testutil-sessions/src/main/kotlin/io/spine/examples/pingh/testing/sessions/given/PredefinedResponses.kt

```diff
@@ -53,3 +53,15 @@ public fun predefinedAccessTokenResponse(): AccessTokenResponse {
     val json = jsonFile.readText()
     return AccessTokenResponse::class.parseJson(json)
 }
+
+/**
+ * Returns the response provided by [PredefinedGitHubAuthenticationResponses]
+ * upon successful execution of the request for refresh access token.
+ */
+public fun predefinedRefreshedAccessTokenResponse(): AccessTokenResponse {
```

Comment:
Please review this file and shorten the names like this one ("predefinedRefreshedAccessTokenResponse").